docs/notes/2.22.x.md

```diff
@@ -24,6 +24,10 @@ If you encounter such discrepancies, and you can't resolve them easily, please [
 
 ### Backends
 
+#### BUILD
+
+Non-parametrized keys in `overrides` or parameter group will now be removed from the target parameters. This fix [a bug](https://github.com/pantsbuild/pants/issues/20933) where a target with parametrized default would have inconsistent parameters with its field values.
```

Comment:
```suggestion
Non-parametrized values in `overrides` will now be removed from the target's address parameters. This fixes [a bug](https://github.com/pantsbuild/pants/issues/20933) where a target with parametrized default values would have inconsistent address parameters with its field values.
```

(removed the "or parameter group", treating it as part of the notion of being "parametrized".)